Server/appsettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Trace",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "ResourceConnectionString": "endpoint=https://web-ui-dev.communication.azure.com/;accesskey=o17eop1mJojDm336WDsbWF3wH+viXxpU5cKI9YvGJ0zmDpbx/eHdhBSEn8CVBHmpctO1Amtm9nn5AnbRdylnWA=="
```

Comment:
remove key (see other comment about potential customers leaking this also! I think we should promote a different practice)

Reply:
Thanks for catching this one. 

There isn't really a good way about it. If we hardcode in a token instead then what if multiple people join?
Until we have a better solution I don't want to brute force anything. The goal of this PR is just to port over the UI Library based hero samples into these repos while still making it easy to port over any additional changes in the future.

Reply:
The alternative is as an environment variable - we can either (or both) in our instructions have:
* set RESOURCE_CONNECTION_STRING environment variable (with `powershell example...`, `bash example...`)
* add a .env file with `RESOURCE_CONNECTION_STRING='<TODO REPLACE ME>'`
  * we could add this .env file, commit it to src code, then add to the gitignore (but that can lead to other confusions)

But yeah definitely lets not block this PR on it but I strongly feel we should change this